### DISCUSSION TITLE
Continuous X-Axis Time rescaling of Live Measurement Data

### DISCUSSION THREAD

**Discussion by Vollbrecht** [grafana/grafana#49355] [Dashboards, Panels & Visualizations]
**What happened**:
Continuous X-Axis Time rescaling of Live Measurement Data.

![grafana_dashboard_scale_err](https://user-images.githubusercontent.com/12041081/168639923-448a4df4-edde-449c-86da-ce0f442a5b72.gif)

**What you expected to happen**:
Stop it from rescaling.

**How to reproduce it (as minimally and precisely as possible)**:
1. Use Live Measurements Query Type that are not to fast like this test one
![Screenshot from 2022-05-16 18-23-40](https://user-images.githubusercontent.com/12041081/168639972-ad19b7b3-fa43-43a0-9b5c-995f33668115.png)

2. Use the Refresh live dashboard option
![Screenshot from 2022-05-16 18-20-20](https://user-images.githubusercontent.com/12041081/168639509-525c6d51-6529-4631-baac-1d88857e2c6a.png)

3. Have multiple pannels (that uses now() in querry?)before the Live Measurements panel on the dashboard stacked such that the Live Measurement panel is not visible on side-loading. In other words one need to scroll down a bit before the Live Measurement panel is visible

**Anything else we need to know?**:

**Environment**:
- Grafana version:
-- 8.5.2 (official docker image)
- Data source type & version:
-- Grafana Live-Measurement / InfluxDBv2 
- OS Grafana is installed on:
--  Docker container grafana/grafana:8.5.2 on debian 11 host
- User OS & Browser:
-- Tested on Linux( Firefox 100 & Chrome 101) and on Android (Firefox 100)
- Grafana plugins:
-- Time series - Native Plugin

**Comment by Vollbrecht** [comment]
Additional information: if one change the time-range settings to an arbitrary value (including the same value as on site loading) after the side is loaded, it stops the continuous rescaling and work as anticipated. But it will break again as soon the side is loaded again.

**Comment by zuchka** [comment]
do you think you could try and replicate this on our public sandbox, https://play.grafana.org? :pray:

**Comment by Vollbrecht** [comment]
I would love to, but it seems the InfluxDB-Flux-Enabled Data Source on play.grafana.org is broken:
[link to minimal broken play.grafana flux querry](https://play.grafana.org/explore?orgId=1&left=%7B%22datasource%22:%22InfluxDB-Flux-Enabled%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22query%22:%22buckets()%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D) 
Is there a working way to use Flux query in conjunction with grafana live measurements on play.grafana.org ?

**Comment by Vollbrecht** [comment]
> I would love to, but it seems the InfluxDB-Flux-Enabled Data Source on play.grafana.org is broken: [link to minimal broken play.grafana flux querry](https://play.grafana.org/explore?orgId=1&left=%7B%22datasource%22:%22InfluxDB-Flux-Enabled%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22query%22:%22buckets()%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D) Is there a working way to use Flux query in conjunction with grafana live measurements on play.grafana.org ?

should i open a separate issue for the Flux query problem in this repository, or is there a different place for wrong play.grafana.org configs

**Comment by zuchka** [comment]
@Vollbrecht ugh, forgive my delay. Your message actually got me taking a closer look at play. I found the configs for play and can confirm that we are currently running Influxdb 1.8 in that backend and that flux is NOT is enabled. So that datasource was never going to work. Someone must have added it to debug a frontend issue or something. I deleted that particular Influx on play. But, I've also started a larger internal conversation to add a second backend: perhaps we could run an Influx v1 with influxQL and v2 with flux enabled or something

**Reply by Vollbrecht** [reply]
OK, thank you for the heads up. If a v2 Database with Flux gets enabled on play.grafana, where can i get notified about it normally ?

**Reply by zuchka** [reply]
perhaps play needs a changelog? I'll try to hop back on this thread if I can get v2 + Flux added 👍

**Comment by Vollbrecht** [comment]
@zuchka i have a working broken dashboard with this issue on my free personal grafana cloud. i tryed to login on play.grafana with github and with an grafana account. They only allow viewing all test dashboards. How can i publish the dashboard to play.grafana? Do i need to reopen a new issue?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
